css-values/Overview.bs

```diff
@@ -1274,16 +1549,55 @@ Angle Units: the <<angle>> type and ''deg'', ''grad'', ''rad'', ''turn'' units</
 -->
 
 <h3 id="time">
-Duration Units: the <<time>> type and ''s'', ''ms'' units</h3>
+Duration units: the <<time>> type and 
+''s'', ''ms'', 
+''min'', ''h'', 
+''now'', ''mom'', 
+'''beat'', ''blink'', 
+''jif'' 
+units</h3>
 
 	Time values are <a>dimensions</a> denoted by <dfn id="time-value">&lt;time></dfn>.
 	The time unit identifiers are:
 
 	<dl export dfn-type=value dfn-for="<time>">
-		<dt><dfn id="s">s</dfn>
+
+		<dt><dfn id="s">s unit</dfn>
 		<dd>Seconds.
-		<dt><dfn id="ms">ms</dfn>
+		
+		<dt><dfn id="ms">ms unit</dfn>
 		<dd>Milliseconds. There are 1000 milliseconds in a second.
+		
+		<dt><dfn id="min">min unit</dfn>
+		<dd>Minutes. There are 60 seconds in a minute.
+		
+		<dt><dfn id="h">h unit</dfn>
+			<!-- hr? -->
+		<dd>Hours. There are 60 minutes in an hour.
+
+		<dt><dfn id="now">now unit</dfn>
+		<dd>Instants. There are 3 seconds in an instant.
+
+		<dt><dfn id="mom">mom unit</dfn>
+		<dd>Moments. There are 90 seconds in a moment.
+
+		<dt><dfn id="beat">beat unit</dfn>
+		<dd>Time between user heart beats. 
+			Depends on the user and situation, longer if relaxed.
+			A beat defaults to 0.8s, 
+			i.e. a pulse rate of 75 beats per minute.
+		
+		<dt><dfn id="blink">blink unit</dfn>
+		<dd>Duration the user's eyelid is closed during a spontaneous blink. 
+			Depends on the user and situation, longer if tired. 
+			A blink defaults to 25 milliseconds.
+		
```

Comment:
A new hardware-dependent unit for optimized animations etc., not yet discussed in an Github issue.